tests/openbb_terminal/core/log/generation/test_formatter_with_exceptions.py

```diff
@@ -0,0 +1,77 @@
+import os
+import pytest
+from openbb_terminal.core.log.generation import formatter_with_exceptions as fwe
+from openbb_terminal.core.log.generation.settings import AppSettings
+from openbb_terminal.core.config.constants import REPO_DIR
+
+app_settings = AppSettings(
+    commit_hash="MOCK_COMMIT_HASH",
+    name="MOCK_COMMIT_HASH",
+    identifier="MOCK_COMMIT_HASH",
```

Comment:
`MOCK_COMMIT_HASH` ?

Reply:
I copied this from what you sent me. Let me know how I should change!